src/less-style/tree-content.less

```diff
@@ -89,10 +89,6 @@
   }
 }
 
-#jstree-marker {
```

Comment:
:+1: there are still references to this inside bower_components in the jstree library. I don't think we reference it directly though